### PR TITLE
Fixes for latest osc client config changes

### PR DESCRIPTION
**Pull request by detiber**
@jwhonce @twiest @wshearn Changes needed for latest build since the default config file locations have changed.

@wshearn this will require a new build of origin in the copr

### CODE REVIEW

**Comment by wshearn** [comment]
New build up in copr

**Comment by detiber** [comment]
@wshearn @twiest this one should be good to go now.
@kwoodson I say next time we enable pylint as a gate on a project, the person enabling must do the initial script cleanup :smiling_imp:

File: roles/openshift_register_nodes/library/kubernetes_register_node.py [hunk 1]
```diff
@@ -1,12 +1,11 @@
 #!/usr/bin/python
 # -*- coding: utf-8 -*-
 # vim: expandtab:tabstop=4:shiftwidth=4
+# pylint: disable=fixme, missing-docstring, too-many-arguments
+# pylint: disable=too-many-locals, too-many-branches, too-few-public-methods
+# pylint: disable=no-self-use
```
Comment (twiest) [inline roles/openshift_register_nodes/library/kubernetes_register_node.py]: Do all of these have to be disabled?

For instance, docstrings are extremely easy to create, and don't require much testing (if any). Can you just quickly fix the docstrings?

And are these pylint tests meant to be permanently disabled or temporarily disabled?

Please add comments saying why they're being disabled, and if it's meant to be temporary or permanent.

File: roles/openshift_register_nodes/library/kubernetes_register_node.py [hunk 2]
```diff
@@ -369,7 +396,7 @@ def main():
             module.fail_json(msg="Unknown error creating node",
                              node=node.get_node())
 
-
+# pylint: disable=redefined-builtin, unused-wildcard-import, wildcard-import
```
Comment (twiest) [inline roles/openshift_register_nodes/library/kubernetes_register_node.py]: same as above. I don't think we should allow pylint disables without a comment explaining why.

**Comment by detiber** [comment]
@twiest you guys are killing me :scream: 

Updated with docstrings and comments

**Comment by detiber** [comment]
@kwoodson: jenkins failure looks related to ansible not being present on the jenkins host
```
+ git/pylint.sh 12ca55504988fe6ea524b222a510d51b0168f95b 0aaa5e6a496ee43f63f3df4831314f705ea57c58 master
************* Module kubernetes_register_node
I: 14, 0: Locally disabling fixme (W0511) (locally-disabled)
I: 14, 0: Locally disabling too-many-arguments (R0913) (locally-disabled)
I: 14, 0: Locally disabling no-self-use (R0201) (locally-disabled)
I: 15, 0: Locally disabling too-many-locals (R0914) (locally-disabled)
I: 15, 0: Locally disabling too-many-branches (R0912) (locally-disabled)
I: 15, 0: Locally disabling too-few-public-methods (R0903) (locally-disabled)
I:597, 0: Locally disabling redefined-builtin (W0622) (locally-disabled)
I:597, 0: Locally disabling unused-wildcard-import (W0614) (locally-disabled)
I:597, 0: Locally disabling wildcard-import (W0401) (locally-disabled)
E:126,22: Undefined variable 'json' (undefined-variable)
E:448,11: Undefined variable 're' (undefined-variable)
E:448,57: Undefined variable 're' (undefined-variable)
E:464,15: Undefined variable 're' (undefined-variable)
E:478,13: Undefined variable 'AnsibleModule' (undefined-variable)
F:599, 0: Unable to import 'ansible.module_utils.basic' (import-error)
Build step 'Execute shell' marked build as failure
Setting status of c2b7fdc7d3ec579e42b752d4a306f71cc5dfaa23 to FAILURE with url https://ci.openshift.redhat.com/jenkins/job/openshift-ansible/21/ and message: Merged build finished.
Finished: FAILURE
```

**Comment by twiest** [comment]
@detiber looks great, thanks for documenting the pylint disables. I think that's a very important requirement if we're going to allow them.

:+1:  going to merge without build bot succeeding because we're still trying to get that worked out. It's not this PRs fault.

**Pull request merged**